### ISSUE TITLE
Dutch Trackers using new DDOS detection not detected as Flaresolverr required

### ISSUE THREAD

**Issue by Visne** [Prowlarr/Prowlarr#916]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Same as with this Jackett issue: https://github.com/Jackett/Jackett/issues/13070.
It is impossible to add some specific Dutch trackers, like Film-Paleis.

### Expected Behavior

Being able to add these trackers.

### Steps To Reproduce

1. Add indexer
2. Choose Film-Paleis
3. Enter username, password, username, baseurl etc
4. Press test or save

### Environment

```markdown
- OS: Manjaro ARM
- Prowlarr: 0.2.0.1448
- Using Reverse Proxy: Yes, Nginx
```

### What branch are you running?

Develop

### Trace Logs?

[prowlarr.trace.txt](https://github.com/Prowlarr/Prowlarr/files/8375559/prowlarr.trace.txt)

[AB#2883](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2883)

**Comment by Visne** [comment]
It's probably the same issue as this one: https://github.com/Jackett/Jackett/issues/13070

**Comment by bakerboy448** [comment]
```
2022-03-30 01:19:34.1|Trace|IndexerHttpClient|Res: [GET] https://www.film-paleis.me/browse.php?search=&incldead=1: 200.OK (10868 ms)
2022-03-30 01:19:34.1|Trace|Cardigann|Attempting to re-auth based on indexer search response
2022-03-30 01:19:34.1|Trace|IndexerHttpClient|Req: [GET] https://www.film-paleis.me/login.php
2022-03-30 01:19:34.1|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-03-30 01:19:34.6|Trace|IndexerHttpClient|Res: [GET] https://www.film-paleis.me/login.php: 200.OK (436 ms)
2022-03-30 01:19:34.6|Trace|IndexerHttpClient|Response content (2033 bytes): <html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>DDoS protection | Film-Paleis.ME</title>
<style type="text/css">
    .main_t {height: 100%; width: 100%; border: 0px solid black;}
    .main_t td {text-align: center;}
    .main_t td.info {height: 100%; text-align: center; vertical-align: top; padding-top: 5em;}
	
    .sign { color: #ccc; margin-top: 5em;}
    .sign a { color: #ccc; }
</style>
<script>
function get_current_url() {
    document.write(window.location.href);
    return null;
}
</script>
</head>
<body>
    <table class="main_t">
        <tr>
            <td class="info">
                <h1>DDoS protection actief voor ip <a href="/blacklists/77.248.174.92" target="_new">77.248.174.92</a></h1>
                <p>Duurt het laden erg lang, klik op de link om door te gaan.</p>
                <a href=''><script>get_current_url();</script></a>


                <p>U word automatisch doorgestuurd na de juiste pagina over 3 seconds.</p>
                <div id=(removed)                    <p>Zorg dat u Cookies en JavaScript heeft ingeschakeld, anders kunt u niet aanmelden.</p>
                </div>
            </td>
        </tr>
        <tr>
            <td class="sign"><span style="color: #49C73B;">
                   DDoS protection by </span><span style="color: #49C73B;">Film-Paleis</span><span style="color: #349EBF;"> Protection</span><br />

        <noscript>Please enable Javascript to see the requested page.</noscript>
            </td>
        </tr>
    </table>
    <script type="text/javascript">
        document.getElementById('js_info').style.display = 'none';
        var date = new Date();

        days = '180';
        date.setTime(date.getTime() + (days * 24*60*60*1000));

        document.cookie = 'ct_anti_ddos_key' + "=" + escape('d2e5ab55f3530a847177596b3e4128e3') + "; expires = " + date.toGMTString() + "; path=/";
        setTimeout(function(){
           window.location.reload(1);
        }, 3000);
    </script>
</body>
</html>



2022-03-30 01:19:34.6|Warn|Cardigann|Unable to connect to indexer

[v0.2.0.1448] NzbDrone.Core.Indexers.Definitions.Cardigann.Exceptions.CardigannConfigException: Login failed: No form found on https://www.film-paleis.me/login.php using form selector form[action="takelogin.php"]
```

Going to need to revisit the Cloudflare detection logic given the new response does not match any known previous responses nor even indicates that we're hitting cloudflare.

**Comment by Visne** [comment]
Well these dutch trackers are all pretty crappy anyways, with horrid web design and weird rules (qBittorrent not allowed for some reason, max ratio limit of 1.5, and having to post a thank you message before being able to download). So I highly doubt they would even know how to implement an API, if they even know what that is.

**Comment by ilike2burnthing** [comment]
Working again.

**Comment by Qstick** [comment]
Closed in e85ccd5808
